### PR TITLE
open external links in new tab

### PR DESCRIPTION
**Pull request by VikramTiwari**
I think this is all, but there might be more such links/

Regarding: https://github.com/GoogleCloudPlatform/gcloud-node/issues/1094

### CODE REVIEW

**Comment by stephenplusplus** [comment]
Thanks for sending this, @VikramTiwari!

@callmehiphop is this the best way to open external links in new windows, or can we use some fancy JS to handle this for us?

File: docs/site/components/language-switcher/language-switcher.html [hunk 1]
```diff
@@ -74,25 +74,25 @@ <h4 class="list-item--heading">External Resources</h4>
       <h4 class="list-item--heading">gcloud Libraries</h4>
     </li>
     <li>
-      <a href="http://googlecloudplatform.github.io/gcloud-java" title="gcloud-java Documentation" class="skip-external-link">
+      <a href="http://googlecloudplatform.github.io/gcloud-java" target="_blank" title="gcloud-java Documentation" class="skip-external-link">
```
Comment (callmehiphop) [inline docs/site/components/language-switcher/language-switcher.html]: These links are supposed to behave more like a dropdown than anchor tags.. so I don't think we want new window here.

**Comment by callmehiphop** [comment]
> is this the best way to open external links in new windows, or can we use some fancy JS to handle this for us?

Well, if we wanted to simplify the whole thing we could create an angular directive to overload the `<a>` tag and set the target dynamically.

**Comment by stephenplusplus** [comment]
I'm cool with that, since a lot of our links are going to be coming from Markdown in the docs.

**Comment by callmehiphop** [comment]
Cool!

@VikramTiwari did you want to take that on? Otherwise I can get to it later :)

**Comment by VikramTiwari** [comment]
I can take a stab at it, but what would be the logic that we would like to use here? Not all absolute links are external links.

**Comment by callmehiphop** [comment]
We use a CSS class called `skip-external-link`, so I would think any absolute URL missing that class would qualify? 

@stephenplusplus thoughts?

**Comment by stephenplusplus** [comment]
Sounds sane to me.

**Comment by stephenplusplus** [comment]
@callmehiphop since all of the docs changes we've pushed, will @VikramTiwari rebasing be enough, or is there anything else to it?

**Comment by callmehiphop** [comment]
Nope :(

We'll probably have to mirror these changes in https://github.com/GoogleCloudPlatform/gcloud-common

**Comment by stephenplusplus** [comment]
@VikramTiwari we moved most of our docs to a centralized repo, so that the other languages (gcloud-ruby, gcloud-php, etc) can set up their docs as well.

The new location is https://github.com/GoogleCloudPlatform/gcloud-common/tree/master/site/src -- I believe the task now is finding links across gcloud-common that need to be updated, as well as some changes still inside this repo. Are you still up for it? It's totally understandable if you'd rather have us do it. Thanks for your help so far and sorry for the curveball!

// @callmehiphop is all of that ^ correct?

**Comment by callmehiphop** [comment]
@stephenplusplus yep!

**Comment by VikramTiwari** [comment]
Hey @stephenplusplus Good move for centralized repo. I will try to do the changes at that repo by this weekend. Will keep you all updated.

**Comment by stephenplusplus** [comment]
Awesome, thanks!

**Comment by stephenplusplus** [comment]
I'll close this up and let #1094 track our progress.

**Pull request closed without merging**